bench/bench.ml

```diff
@@ -90,32 +90,85 @@ let prepare_workspace () =
       Format.eprintf "cloning %s/%s@." pkg.org pkg.name;
       Package.clone pkg)
 
-let dune_build () =
+let dune_build ?(suffix = "") () =
```

Comment:
Instead of suffix, what about a mandatory argument called "name"

Reply:
done